src/content/tools/pub/cmd/pub-lish.md

```diff
@@ -36,5 +36,14 @@ In the event of warnings, your package *is* uploaded.
 To ensure that your package has no warnings before uploading,
 either don't use `--force`, or use `--dry-run` first.
 
+### `--skip-validation`
+
+Publishes without going through the client-side validation process or resolving dependencies.
+This is useful for advanced users who knows why the validation might fail and wishes to side step a particularly issues.
```

Comment:
**suggestion:** This needs semantic breaks and active voice.

```suggestion
To side step a particular issue, you might want to use this option.
Advanced users might know why the validation might fail,
but know the issue would be resolved in other related work.
```

Reply:
I updated it to:

> This is useful for advanced users who knows why the validation fails and wishes to side step a particularly issues.

I can see how it makes sense to write:
> To side step a particular issue, you might want to use this option.

But I kind of wanted to suggest that you should only do this if you are an advanced user and you know what you're doing. And know what particular issue you're side stepping.

The following appears to me like a statement about the knowledge that an "advanced user" may or may not have.

> Advanced users might know why the validation might fail,
> but know the issue would be resolved in other related work.

I don't understand it.